doc/source/WorkingWithScenes/AnatomyOfACamera/index.md

```diff
@@ -0,0 +1,136 @@
+
+```eval_rst
+.. figure:: images/illustrationDepthOfField.png
+    :scale: 100%
+    :alt: Depth of field blur in Gaffer
+
+    **Figure 3.** An approximation of depth of field blur in Gaffer.
+```
+
+> Important :
+> The term _lens aperture_ has a meaning near to its real-world counterpart – it really can be thought of as a round opening in space.
+
+The fStop and focalDistance parameters are identical to f-number and focus distance in real lenses. They allow easy replication of real lens settings.
+
+The third parameter, focalLengthWorldScale, sets the scale between the camera's unit of measure and the world space's unit of measure. Because the circle occupies 3D space, the circle must be measured in world space units. Complicating things is the fact that the world space units might not be measured in millimeters. The scene could use any scale – centimeter, meter, foot, kilometer, etc. Therefore, the scale factor between them must be defined.
+
+For depth of field, lens aperture is calculated with the formula: 
+
+```
+lens aperture = (focalLength × focalLengthWorldScale) / fStop
+```
+
+For example, assume a lens with a focalLength of 50mm, an fStop of 4, and a world space measured in centimeters (default for Alembic/USD). First, we need to scale the focal length from mm to cm. Using a focalLengthWorldScale of 0.1 yields `50mm × 0.1 = 0.5cm`. This is the lens aperture at its widest. Dividing by the fStop of 4 results in the stopped-down lens aperture of `0.5cm / 4 = 0.125cm`.
+
+
+## Cameras in the Scene ##
+
+
+### Camera data ###
+
+Within the <a href="../AnatomyOfAScene/index.html#scene-hierarchy">scene paradigm</a>, a camera, just like any other scene component, starts with a location in the scene hierarchy. To define a camera, the location must have the following data:
+
+- **Transform:** The vectors that define the position and orientation of the camera.
+- **Object:** A special camera object at the location. Instead of geometry, the object stores camera data, called parameters.
+    - **Parameters:** The crucial values that define a camera, such as the perspective type, field of view/aperture, and depth of field settings. If defined, a special kind of optional parameters, called **render overrides**, will supercede the scene's <a href="../AnatomyOfAScene/index.html#options">render options</a> during computation and rendering.<br>
+    ![Camera parameters in the Scene Inspector](images/interfaceCameraParameters.png)
+- **Sets:** A list of sets the location belongs to. By default, every camera is assigned to an automatic "Cameras" set, accessible in the API by the `__cameras` variable.<br>
+    ![Camera sets in the Scene Inspector](images/interfaceCameraSets.png)
+
+
+### Data flow ###
```

Comment:
@johnhaddon You'll definitely want to take a look at this section, especially with the accompanying figure.

Reply:
I may not be the right audience, but I found the diagram a little confusing, specifically in the interplay between overrides on the left and the globals on the right. To me, the red dotted lines and strike-throughs in the StandardOptions section imply some sort of "computational interplay" between the location and the globals at that point in the graph, which isn't how it is at all. The location and the globals just merrily go down through the graph totally independently, only being resolved into the final render settings when we perform a render. That "override resolution" isn't part of the computation engine at all, but something we do when translating the scene to the renderer.

I'll make a few specific comments below with thoughts on how this might be improved from my perspective, but it might be that for other audiences it's OK anyway.

Reply:
> The location and the globals just merrily go down through the graph totally independently, only being resolved into the final render settings when we perform a render.

This is how I originally arranged the figure and explanation, however this will likely confuse hands-on users who notice that when selecting the look-through camera in the _Viewer_ mid-graph, overrides and tweaks are taken into account. As far as I know, computationally, the OpenGL renderer is as much a renderer as Arnold, but I suspect users don't think of it this way. Using language like "final render," or anything that conveys finality, might suggest the necessity of render task+StandardOptions nodes for the overrides to apply, which users will find isn't the case.

I agree that the red arrows can be (falsely) interpreted as data flow. The other obvious solution is to insert asterisks next to the tweaked parameters and the render options, and add a footnote explaining what's happening, but it would have added more text in an already text-rich illustration. I'll see if there's another way to convey it.

Reply:
> Using language like "final render," or anything that conveys finality, might suggest the necessity of render task+StandardOptions nodes for the overrides to apply, which users will find isn't the case.

Good point. I do want to make sure we avoid giving the wrong mental model though, and I'm not sure we're there yet. Let me take a stab at an alternate introductory paragraph :

"""
Like geometry, cameras are represented as objects at locations in the scene hierarchy. But to look through a camera (either in the Viewer, or a render), Gaffer needs additional settings such resolution and film fit. These additional settings are specified via the scene globals, and combined with the camera at the point of use. Here we describe this dataflow, and show how cameras can provide optional overrides for the scene globals.
"""

Reply:
Incorporated your paragraph, with some minor changes here and there. How does it look now?